src/rpc/misc.cpp

```diff
@@ -293,6 +294,7 @@ UniValue createmultisig(const JSONRPCRequest& request)
             "       \"key\"    (string) bitcoin address or hex-encoded public key\n"
             "       ,...\n"
             "     ]\n"
+            "3. \"fSort\"      (bool, optional) Whether to sort public keys according to BIP67. Default setting is false.\n"
```

Comment:
It should be a boolean. Just observed they aren't usually quoted in RPC output, fixing now.